### PR TITLE
[ticket/15298] Makes sure CLI exceptions are never suppressed by Symfony

### PR DESCRIPTION
**Pull request by Nicofuma**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [ ] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15298

### CODE REVIEW

File: phpBB/phpbb/files/filespec.php [hunk 1]
```diff
@@ -394,7 +394,8 @@ public function check_content($disallowed_content)
 	 *
 	 * @param string $destination Destination path, for example $config['avatar_path']
 	 * @param bool $overwrite If set to true, an already existing file will be overwritten
-	 * @param bool $skip_image_check If set to true, the check for the file to be a valid image is skipped
+	 * @param bool $skip_image_check If set to true, the check for the file to
+	 *                               be a valid image is skipped
```
Comment (Elsensee) [inline phpBB/phpbb/files/filespec.php]: Unrelated change?

Comment (Nicofuma) [reply phpBB/phpbb/files/filespec.php]: I don't know where it is coming from, I didn't touch the file....

**Comment by Nicofuma** [comment]
@marc1706 should be ready

**Comment by Nicofuma** [comment]
@marc1706 up

**Comment by marc1706** [comment]
Rebased and updated to latest master :)
